support/openharmony/.gitignore

```diff
@@ -0,0 +1,4 @@
+# ignore generated files
+.hvigor
+# ignore secrets
+signing_config.json
```

Comment:
the signing config json can be at an arbitrary user provided path, so we can't really ignore the file reliably. Doesn't harm to add the ignore this file of course though.